challenge.rb

```diff
@@ -44,8 +52,16 @@ def okay_to_drink? age, drinking_age
 # If the AC is non-functional and it's NOT too hot, `ac_monitor` should return
 #   "Fix the AC whenever you have the chance. It's cool."
 
-def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+def ac_monitor (current_temp, ac_working, desired_temp)
+  if (ac_working == "yes" ) && (current_temp > desired_temp)
+    puts "Turn on the AC."
+  elsif (ac_working == "yes") && (current_temp < desired_temp)
+    puts "Just right!"
+  elsif (ac_working == "no") && (current_temp > desired_temp)
+    puts "Fix the AC now! It's hot!"
+  elsif (ac_working == "no") && (current_temp < desired_temp)
+    puts "Fix the AC whenever you have the chance. It's cool."
+  end
```

Comment:
`return`, not print. In fact, since Ruby implicitly returns the last evaluated expression, this could even have been written as 
```
if (ac_working == "yes" ) && (current_temp > desired_temp)
    "Turn on the AC."
elsif (ac_working == "yes") && (current_temp < desired_temp)
    "Just right!"
elsif (ac_working == "no") && (current_temp > desired_temp)
    "Fix the AC now! It's hot!"
elsif (ac_working == "no") && (current_temp < desired_temp)
    "Fix the AC whenever you have the chance. It's cool."
end
```
Otherwise right, though.